src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -394,13 +407,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
   /** Get all inputs for the scope of this call. */
   override def getInputs(workflowId: WorkflowId, call: Call): Future[Traversable[SymbolStoreEntry]] = {
     require(call != null, "call cannot be null")
-    getSymbols(workflowId, IoInput, Option(call))
+    getSymbols(workflowId, IoInput, Option(call.fullyQualifiedName))
   }
 
   /** Get all outputs for the scope of this call. */
-  override def getOutputs(workflowId: WorkflowId, call: Call): Future[Traversable[SymbolStoreEntry]] = {
-    require(call != null, "call cannot be null")
-    getSymbols(workflowId, IoOutput, Option(call))
+  override def getOutputs(workflowId: WorkflowId, callFqn: FullyQualifiedName): Future[Traversable[SymbolStoreEntry]] = {
+    require(callFqn != null, "callFqn cannot be null")
```

Comment:
It'd be good to avoid `null` as much as possible even if to stay out of the habit. I'll note that `Option(null)` is automatically a `None`, perhaps we could do something w/ that here.